Model/Config/FastIntegrationConfig.php

```diff
@@ -33,7 +33,7 @@ class FastIntegrationConfig
     const XPATH_FAST_PROD_JS_URL = 'fast_integration/fast/fast_prod_js_url';
     const XPATH_ENABLE_DARK_THEME = 'fast_integration/fast/enable_dark_theme';
     const XPATH_ENABLE_DEBUG_LOGGING = 'fast_integration/fast/enable_debug_logging';
-    const XPATH_ENABLE_REST_API_LOG = 'fast_integration/fast/enable_rest_api_log';
+    const XPATH_ENABLE_REST_API_LOG = 'fast_integration/rest_api_log/enable_rest_api_log';
```

Comment:
Should this conform to /fast/ as above ?

Reply:
No, that was the issue previously. The `enable_rest_api_log` setting is in the `rest_api_log` group, so this update fixes the key.